src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -70,11 +70,14 @@ std::string postEntities
 
```

Comment:
No, nothing missing.
As you know, we don't really touch ciP->httpStatusCode inside mongoBackend.
This is just like a security measuremente. No known bugs are fixed + 'make test' worked just fine ...

NTC